services/web-server/src/clients.js

```diff
@@ -1,6 +1,5 @@
 const {
   Auth,
-  AwsProvisioner,
   EC2Manager,
```

Comment:
This, too, is no longer used (here or in `src/loaders/workerTypes.js`)

Reply:
Thanks! I updated this in both places.